app/beachbot_controller.py

```diff
@@ -1,7 +1,20 @@
-# the future is now...
+# the future is now... (avoids printing pytoch warnings about deprecated functions to console)
```

Comment:
?? A comment at the top of this file suppresses pytorch warnings? Why would we want to suppress warnings in the first place? We should fix them at some point, so hiding them will only hurt us.